app/assets/javascripts/components/utils.js.coffee

```diff
@@ -132,3 +132,9 @@ class @Utils
       .fail (xhr, status, error) ->
         alert('Error: ' + error)
         callback()
+
+  @select2TagClickHandler: (e, elem) ->
+    if e.which == 1
```

Comment:
I thought about that, but currently you also lose the changes when you click on "Back" or a link in the navigation, opening the links in a new window also defeats the purpose a bit. The idea was to be able to quickly navigate through an agent network when you notice you opened the wrong Agent.
If the user clicks on a link by accident the browser back button works well, I was able to get back the form with all the changes I made before leaving the page.